src/Meilisearch/Index.cs

```diff
@@ -470,25 +423,25 @@ public async Task<IEnumerable<string>> GetDisplayedAttributesAsync(CancellationT
         /// </summary>
         /// <param name="displayedAttributes">Collection of displayed attributes.</param>
         /// <param name="cancellationToken">The cancellation token for this call.</param>
-        /// <returns>Returns the updateID of the asynchronous task.</returns>
-        public async Task<UpdateStatus> UpdateDisplayedAttributesAsync(IEnumerable<string> displayedAttributes, CancellationToken cancellationToken = default)
+        /// <returns>Returns the task Uid of the asynchronous task.</returns>
```

Comment:
```suggestion
        /// <returns>Returns the task info of the asynchronous task.</returns>
```
All the settings function said it's return task Uid but I think it returns task info no?